project_details.gs

```diff
@@ -0,0 +1,19 @@
+/**
+* Get the details of the project
+* @param {String projectId of the GCP project
+* @return {Object} Project Details having name, Project Number, project Id and other details
```

Comment:
your comment and functions names are not matching. Function stats that you are just retrieving project number. Also can you rename variable from projectId to cloudProjectId and so on

Reply:
Updated the function name and variable name